views/partials/head.ejs

```diff
@@ -10,27 +10,30 @@
 
 </head>
 <body>
-
     <header>
         <div>
             <a href="/">
-                <img src="../assets/logo\RedPers_Logo_Cmyk_Black.webp" 
-                alt="Red Pers Logo"
-                width="300" 
-                >
+                <img src="../assets/logo/RedPers_Logo_Cmyk_Black.webp" 
+                     alt="Red Pers Logo"
+                     width="300">
             </a>
-            <nav role="navigation">
-                <input type="checkbox">
-                <ul id="nav-menu">
-                    <li><a href="">Binnenland</a></li>
-                    <li><a href="">Buitenland</a></li>
-                    <li><a href="">Columns</a></li>
-                    <li><a href="">Economie</a></li>
-                    <li><a href="">Kunst & Media</a></li>
-                    <li><a href="">Podcasts</a></li>
-                    <li><a href="">Politiek</a></li>
-                    <li><a href="">Wetenschap</a></li>          
+            <nav id="navbar-toggle">
+                <input type="checkbox"/>
+                <span class="navbar-menu-button"></span>
+                <span class="navbar-menu-button"></span>
+                <span class="navbar-menu-button"></span>
+                <ul class="navbar-menu">
+                    <li><a href="#"><nobr>Binnenland</nobr></a></li>
+                    <li><a href="#"><nobr>Buitenland</nobr></a></li>
+                    <li><a href="#"><nobr>Columns</nobr></a></li>
+                    <li><a href="#"><nobr>Economie</nobr></a></li>
+                    <li><a href="#"><nobr>Kunst & Media</nobr></a></li>
+                    <li><a href="#"><nobr>Podcasts</nobr></a></li>
+                    <li><a href="#"><nobr>Politiek</nobr></a></li>
+                    <li><a href="#"><nobr>Wetenschap</nobr></a></li>  
```

Comment:
Het _nobr_ element is deprecated

Reply:
https://github.com/SamaraFellaDina/pleasurable-ui/commit/7162c1ae8a872ee7f033b57a5365a2f1844803c0

Aanpast!